### PR TITLE
Initial report structure and table of contents

### PR DESCRIPTION
**Pull request by ciprianprohozescu**

### CODE REVIEW

File: sections/chapters/data_discovery_tool.tex [hunk 1]
```diff
@@ -0,0 +1,11 @@
+\chapter{The Data Discovery Tool}\label{ch:ch2label}
+
+\section{Discovery Strategy}
+
+\section{The Technology Stack}
+
+\section{Infrastructure}
+
+\section{Scalability}
```
Comment (BalazsAgardi) [inline sections/chapters/data_discovery_tool.tex]: Isn't Infrastructure and scalability linked? How do you distinguish between the technology stack and infrastructure?

**Review by BalazsAgardi — COMMENTED** [review]
- Both Danish and English logo shows up for me
- Blank page after title page
- copyright info both English and Danish; separate page?

File: sections/chapters/data_discovery_tool.tex (hunk 1 above)
Comment (ciprianprohozescu) [reply sections/chapters/data_discovery_tool.tex]: That's a valid point. These chapters are just some ideas, that's why I said I need help with brainstorming more. And change these ones if needed.

**Comment by ciprianprohozescu** [comment]
> - Both Danish and English logo shows up for me
> - Blank page after title page
> - copyright info both English and Danish; separate page?

Martinus please check that these happen for you. I can't see any of them when I compile on my machine.

**Pull request merged**